legion/engine/rendering/data/material.cpp

```diff
@@ -149,7 +149,7 @@ namespace legion::rendering
 
     void material::make_unsavable()
     {
-        m_canLoadOrSave = false;
+        //m_canLoadOrSave = false;
     }
```

Comment:
is it possible to just delete this function and the existence of `m_canLoadOrSave `?